app/loader.cpp

```diff
@@ -230,6 +274,11 @@ bool Loader::layerVisible( QgsMapLayer *layer )
   return false;
 }
 
+QString Loader::qgisLog()
+{
+  return mQgisLog;
```

Comment:
Can we rename this `mQgisLog` to something like `mProjectLoadingLog` so that we know in future what it contains?